test/Log/AccessLogFactoryTest.php

```diff
@@ -19,7 +19,6 @@
 use Psr\Log\LoggerInterface;
 use ReflectionProperty;
 use Webmozart\Assert\Assert;
-use Zend\Expressive\Swoole\Log\AccessLogFormatterInterface as LegacyAccessLogFormatterInterface;
```

Comment:
This is removed, as we conflict with the zend-expressive-swoole package now, so the two cannot be installed in parallel.